src/lib/Member.svelte

```diff
@@ -7,8 +7,19 @@
     {#each data.members as member}
         <article class="card">
             <picture>
-                <img src="https://fdnd-agency.directus.app/assets/{member.photo}?format=avif"
-                     type="image/avif" width="{member.photo}" height="{member.photo}" alt="{member.photo}">
+                {#if member.photo}
+                    <source srcset="https://fdnd-agency.directus.app/assets/{member.photo}?format=avif" type="image/avif"
```

Comment:
goed dat je picture en srcset gebruikt. Ik dacht dat de if statement om de hele picture element heen moest. Als het werkt en verder nergens issues geeft is dit ook prima.